docs/en_us/internal/testing.rst

```diff
@@ -94,6 +94,22 @@ Test Locations
    -  Bok Choy Accessibility Tests: located under ``common/test/acceptance/accessibility``
    -  Bok Choy PageObjects: located under ``common/test/acceptance/pages``
 
+-  Content type examples:
+
+   The Accessibility Mega Course
```

Comment:
I would prefer to make the name of the course the link text, rather than make the URL the link text.  For screen reader users, it's quite annoying to hear the URL read out loud.  The name we ended up with was "Demo Testing Course" which I think is slightly better since we are not limited to just accessibility testing here.